builds/e2e/e2e.yaml

```diff
@@ -16,15 +16,26 @@ resources:
       - master
       - release/*
 
+variables:
+  # A 'minimal' pipeline only runs one end-to-end test (TempSensor). This is useful for platforms or
+  # environments that are very similar to other platforms/environments in our matrix, e.g. Windows
+  # Server 2019 vs. Windows 10 Enterprise, or Ubuntu 18.04 with the 'docker-ce' package vs. Ubuntu
+  # 18.04 with the 'iotedge-moby' package vs. the same variations in Ubuntu 20.04. In these
+  # instances the platforms/environments are so similar that we don't reasonably expect to encounter
+  # differences--if we do, it would likely manifest during installation, or in running a very basic
+  # test. We don't need to repeat the entire test suite.
+  # The 'minimal' variable defaults to 'false'; we override it in specific jobs as needed.
+  minimal: false
+
 jobs:
 
-################################################################################	
+################################################################################
   - job: linux_arm32v7
-################################################################################	
+################################################################################
     displayName: Linux arm32v7
 
     pool:
-      name: $(pool.name)
+      name: $(pool.custom.name)
```

Comment:
How are these pool created? and config? 
Do you have an internal document I could step through and spin some up for a personal use? (It would be a fun exercise).

Reply:
I have some internal documentation I can share with you. There isn't much use in documenting the internal details in this repo, although it would be nice to eventually move the agent configuration here (e.g., the manifests/scripts that show what base OS and prerequisite software we install on our agents).